tests/api/test_auth.py

```diff
@@ -16,21 +14,8 @@ def test_private_endpoint_no_auth(self):
     json_response = json.loads(response.data.decode("utf-8"))
 
     self.assertEqual(json_response["description"], "Authorization header is expected")
-  
-  @unittest.skipIf(not os.environ.get("AUTH0_CLIENT_ID") or not os.environ.get("AUTH0_CLIENT_SECRET"),
-                     "Cannot run test without AUTH0_CLIENT_ID and AUTH0_CLIENT_SECRET environment variables")
-  def test_private_endpoint_auth(self):
-    data = {
-      "client_id": os.environ.get('AUTH0_CLIENT_ID'),
-      "client_secret": os.environ.get('AUTH0_CLIENT_SECRET'),
-      "audience": API_AUDIENCE,
-      "grant_type":"client_credentials"
-    }
-
-    jwt_response = requests.post(f'https://{AUTH0_DOMAIN}/oauth/token', data=data)
-    jwt = json.loads(jwt_response.text)['access_token']
-    
-    headers = {'Authorization': f'Bearer {jwt}'}
```

Comment:
I abstracted this functionality into `test_utils` so it could be reused between tests.